lib/web_ui/test/engine/platform_dispatcher_test.dart

```diff
@@ -60,9 +60,8 @@ void testMain() {
     test('responds correctly to flutter/platform Clipboard.getData failure',
         () async {
       // Patch browser so that clipboard api is not available.
-      final dynamic originalClipboard =
-          // ignore: implicit_dynamic_function
-          js_util.getProperty(html.window.navigator, 'clipboard');
+      final Object? originalClipboard =
+          js_util.getProperty<Object?>(html.window.navigator, 'clipboard');
```

Comment:
`getJsProperty`?

Reply:
Tests are exempt from JS access policies. They can do whatever they want. However, if you see an advantage in using `safe_browser_api.dart`, I can change tests too.

Reply:
Nah, you're good.